misk-admin/src/main/kotlin/misk/web/v2/Wrapper.kt

```diff
@@ -0,0 +1,11 @@
+package misk.web.v2
+
+import kotlinx.html.TagConsumer
+import misk.tailwinds.TailwindsHtmlWrapper
+import misk.turbo.addHotwireHeadImports
+
+fun TagConsumer<*>.Wrapper(appRoot: String, title: String, playCdn: Boolean, headBlock: TagConsumer<*>.() -> Unit = {}, bodyBlock: TagConsumer<*>.() -> Unit) {
```

Comment:
not sure i understand this

Reply:
`TagConsumer<*>` is the base class that most of the kotlinx.html DSL is extension functions on top of. https://github.com/Kotlin/kotlinx.html

The Wrapper sets up the base html and head blocks that every page needs.